### PR TITLE
feat(tooltip): prefix change

### PR DESCRIPTION
**Pull request by mJarsater**
**Describe pull-request**  
Update the component name and updated the prefix.

**Solving issue**  
Fixes: https://tegel.atlassian.net/browse/DTS-1653

**How to test**  
1. Go to storybook
2. Check in Components -> Tooltip
3. Check the component API and styles.

### CODE REVIEW

**Comment by timrombergjakobsson** [comment]
Readme missing?

**Comment by theJohnnyMe** [comment]
@mJarsater  - I pushed commit for a small typo in the story file that was there from before. Could not target it in the review tool and felt it is too small to bother you around it. Please let me know if that does not feel OK with you.

**Review by theJohnnyMe — APPROVED** [review]
LGTM

File: core/src/components/tooltip/tooltip.stories.tsx [hunk 1]
```diff
@@ -55,7 +55,7 @@ export default {
     },
     text: {
       name: 'Tooltip text',
-      description: 'Sets the sext that will be displayed inside Tooltip.',
+      description: 'Sets the text that will be displayed inside Tooltip.',
       control: {
```
Comment (mJarsater) [inline core/src/components/tooltip/tooltip.stories.tsx:59]: Lol, nice one!

**Pull request merged**